mockfirestore/main.py

```diff
@@ -79,6 +85,15 @@ def create_time(self) -> Timestamp:
         timestamp = Timestamp.from_now()
         return timestamp
 
+    @property
+    def update_time(self) -> Timestamp:
+        return self.create_time
+
+    @property
+    def read_time(self) -> Timestamp:
+        timestamp = Timestamp.from_now()
+        return timestamp
```

Comment:
These don't seem to be present on the original `DocumentSnapshot`. What's the intention with adding them here?

Reply:
They are undocumented properties. You can see them mentioned in the docs in the initialiser for ‘DocumentSnapshot’. We use those properties to sort documents by update time, so we need to have them mocked here.

Reply:
So they are. Fair enough then.